org.kde.spectacle.json

```diff
@@ -13,7 +13,7 @@
         "--filesystem=xdg-pictures",
         "--talk-name=org.kde.kglobalaccel",
         "--talk-name=org.kde.plasmashell",
-        "--talk-name=org.kde.kwin",
+        "--talk-name=org.kde.KWin",
```

Comment:
This is weird. Was the name incorrect or was it get changed?

Reply:
was incorrect